src/styles.css

```
@@ -871,6 +939,7 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
   color: var(--orange-deep);
   background: var(--orange-pale);
 }
+}
```

Stylelint (17.12.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)